resources/translations/fi.edn

```diff
@@ -92,6 +92,7 @@
                    :license-name "Lisenssin nimi"
                    :licenses "Lisenssit"
                    :modifier "Muokkaaja"
+                   :more-info "Lisätietoja-URL (vapaaehtoinen, puuttuessa käytetään resurssin URN:ää)"
```

Comment:
Isn't it so that URN is used only if there is no link and resource id looks like a URN? Otherwise no link.

Reply:
yeah

Reply:
Maybe the text is slightly misleading now. "vapaaehtoinen, puuttuessa kokeillaan URN"?

Reply:
I'd rather not try to translate a complex sentence into swedish, but if you insist... :grimacing:

Reply:
This could really use some info text help or link to manual.

Reply:
Let's keep the description for now.